DependencyInjection/Configuration.php

```diff
@@ -68,6 +68,17 @@ public function getConfigTreeBuilder()
                             ->end()
                         ->end()
                     ->end()
+                    ->children()
+                        ->arrayNode('symmetric')
+                            ->useAttributeAsKey('name')
+                            ->prototype('array')
+                                ->children()
+                                    ->scalarNode('path')->isRequired()->end()
+                                    ->booleanNode('base64')->isRequired()->defaultTrue()->end()
```

Comment:
May be, but i'm not sure that "output" is a right word for both encryptor and decryptor: for the first of them this option applies to result (i.e. output), but for the second - to argument (i.e. input).